c2corg_ui/templates/helpers/view.html

```diff
@@ -6,6 +6,61 @@
     from c2corg_ui.views.document import get_slug
 %>
 
+<%def name="get_licence(doc)">\
+<%
+  doctype = doc['type']
+  CC_ND_NC = False
+  CC_BY_SA = False
```

Comment:
Actually we have cases where images are under "copyright" licence, not creative commons => book covers for instance. See https://github.com/c2corg/v6_common/blob/master/c2corg_common/attributes.py#L576-L580